packages/experimental/Menu/src/MenuPopover/useMenuPopover.ts

```diff
@@ -48,6 +48,11 @@ export const useMenuPopover = (_props: MenuPopoverProps): MenuPopoverState => {
     setPopoverHoverOutTimer(timer);
   }, [openOnHover, setOpen, setPopoverHoverOutTimer]);
 
+  const [canFocusOnContainer, setCanFocusOnContainer] = React.useState<boolean>(shouldFocusOnContainer);
+  const onBlur = React.useCallback(() => {
+    setCanFocusOnContainer(false);
```

Comment:
nit, can we name this Popover instead of container. I was very much not a fan of the "container" name from the get go

Reply:
Yeah, just renamed.

Reply:
Approved without testing on macOS because... well I can't keep doing that. I think it should be OK though, I'm not even sure we expose that prop.